components/profile/layout/MissingNameModal/MissingNameModal.tsx

```diff
@@ -0,0 +1,62 @@
+import { useContext, useState } from "react";
+import { DonorContext } from "../donorProvider";
+import { Lightbox } from "../../../shared/components/Lightbox/Lightbox";
+import { EffektTextInput } from "../../../shared/components/EffektTextInput/EffektTextInput";
+import { saveDonor } from "../../_queries";
+import { useAuth0 } from "@auth0/auth0-react";
+import { AlertCircle, Check } from "react-feather";
+import { toast } from "react-toastify";
+
+import styles from "./MissingNameModal.module.scss";
+
+export const MissingNameModal: React.FC = () => {
+  const { donor, setDonor } = useContext(DonorContext);
+  const { user, getAccessTokenSilently } = useAuth0();
+  const [name, setName] = useState("");
+  const [loading, setLoading] = useState(false);
+  const [open, setOpen] = useState(donor?.name === "");
```

Comment:
Yes indeed! I missed that because for some reason I thought this was guaranteed by a conditional render higher up the chain, but when double checking, it doesn't seem like it. Nice catch 👍